### PR TITLE
Link CoC in README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by denschub**
(No issue, following up #1660.)

I realized the CoC was not linked anywhere, albeit [requested](https://github.com/webcompat/webcompat.com/pull/1660#pullrequestreview-50354745), so I went ahead and added a link in both `README.md` and `CONTRIBUTING.md`. In addition, I did a bold and renamed `codeofconduct.md` to `CODE_OF_CONDUCT.md` since that's a bit of a [GitHub convention](https://help.github.com/articles/adding-a-code-of-conduct-to-your-project/).

r? @miketaylr

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ You are welcome to contribute to this project. Here are the guidelines we try to
 * [Production Server Setup](docs/prod-server.md)
 * [Acknowledgements](#acknowledgements)
 
+Please note that everyone interacting in our codebases, issue trackers, and chat rooms is expected to follow our [code of conduct](https://github.com/webcompat/webcompat.com/blob/master/CODE_OF_CONDUCT.md) so we can all enjoy the effort we put into this project.
```
Comment (zoepage) [inline CONTRIBUTING.md]: How about we change `chat rooms` to `other form of interaction e.g. chat rooms or mailing lists`?

Comment (denschub) [inline CONTRIBUTING.md]: You are very right, changed that!

Comment (zoepage) [inline CONTRIBUTING.md]: Thank you very much!

**Comment by miketaylr** [comment]
Thank you!

**Pull request merged**